src/lib/Sympa/Request/Handler/include.pm

```diff
@@ -280,7 +292,7 @@ sub _twist {
         $request, @result{qw(added deleted updated)}
     );
     $self->add_stash($request, 'notice', 'include_performed',
```

Comment:
This notice will be shown regardless of the outcome of the inclusion:

```
sympa@local-sympa:/var/lib/sympa/list_data/lists.example.com/test$ /usr/local/sympa/bin/sympa.pl --sync_include=test@lists.example.com
include [notice] Including subscribers from data source "subscribers.txt" into list test failed.
include [notice] Inclusion succeeded (0 added, 0 deleted, 0 updated).
```

Reply:
Yes.  If multiple data sources are configured for a single list, some may succeed and some may fail.  `include_performed` report simply informs that inclusion task has been finised. What is more suitable text? "Inclusion finished"?

Reply:
I separated `include_incomplete` and `include_performed`. See cd429cb.